AENS.md

```diff
@@ -90,7 +90,7 @@ interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 ### AENS Entry
 
-This is the entry as it should be stored by a node.
+This is the **top** name entry as it should be stored by a node.
```

Comment:
I read above:

> Any label that is not at the top level, e.g. `aet`, ...

So a "top" name is e.g. "aet".

So this paragraph refers to names without any dots in them?

Reply:
My understanding is that "aet" or "test" are called "registrars" in the document. 

"Any label..." - I'm not entirely sure whether I should correct things already in document or focus on subnames description. I think it makes sense to restrict ourselves to subnames description here, and possibly make another correction PR which could focus on fixing ambiguities in the whole text.